pkgs/by-name/te/tectonic/package.nix

```diff
@@ -53,6 +53,14 @@ symlinkJoin {
 
   meta = tectonic.unwrapped.meta // {
     description = "Tectonic TeX/LaTeX engine, wrapped with a compatible biber";
+    longDescription =
+      (tectonic.unwrapped.longDescription or "")
+      + ''
+        Note that:
+        - `tectonic` by default provides `tectonic` bundled with the compatible `biber`
+        - `tectonic.biber` is the compatible `biber` executable
+        - `tectonic.unwrapped` provides the unwrapped `tectonic` package
+      '';
```

Comment:
Hmm OK I guess this is good, but it is slightly confusing that in a search you'd encounter both the mention of `tectonic.{biber,unwrapped}` here and (with or without aliases enabled) also `biber-for-tectonic` and `tectonic-unwrapped`? You'd encounter them both in `nix search` and search.nixos.org . Since `nix search` doesn't show `longDescription`s but does show aliases (by default), and search.nixos.org doesn't show aliases but does show `longDescription`s, perhaps we should go after all with an alias, even with a warning?

Reply:
Indeed there is some inconsistency here... Maybe we simply drop this `longDescription`? This now just adds noise to the consumer. With the `pkgs/by-name/` stubs we can achieve full backward compatibility, so we just keep the long names (`tectonic-unwrapped`, `biber-for-tectonic`) as "public interfaces"?

Reply:
> Indeed there is some inconsistency here... Maybe we simply drop this `longDescription`? This now just adds noise to the consumer. With the `pkgs/by-name/` stubs we can achieve full backward compatibility, so we just keep the long names (`tectonic-unwrapped`, `biber-for-tectonic`) as "public interfaces"?

OK that's fine with me :).